src/common/playlists/FeaturedPlays.jsx

```diff
@@ -1,22 +1,21 @@
 import PlayThumbnail from "./PlayThumbnail";
-
-import { getFeaturedPlays } from "meta/play-meta-util";
+import useFeaturedPlays from "common/hooks/useFeaturedPlays";
 
 const FeaturedPlays = () => {
-  const plays = getFeaturedPlays();
-  console.log(plays);
+  const [loading, error, data] = useFeaturedPlays();
 
   return (
     <>
-      <h3 className="plays-title-primary">
-          Trending <strong>Plays</strong>
+      <h3 className='plays-title-primary'>
+        Trending <strong>Plays</strong>
       </h3>
-      <ul className="list-plays">
-        {plays.map((play, index) => (
+      <ul className='list-plays'>
+        {data?.map((play, index) => (
```

Comment:
Please do the same empty and undefined check.

Also use the loading...

Reply:
@Angryman18 please take care

Reply:
handled.